src/cloud/project/project-conf-files_magento-app.md

```diff
@@ -260,7 +260,7 @@ dependencies:
 hooks:
     build: |
         cd public/profiles/project_name/themes/custom/theme_name
-        npm install
+        npm install --prefer-offline --no-audit --no-optional
```

Comment:
Build environments on cloud aren't (necessarily) saved across build runs, and looking at some [published benchmarks](http://www.tiernok.com/posts/2019/faster-npm-installs-during-ci/) it seems that using `npm ci` would provide provide much better performance than any of these other flags.